### ISSUE TITLE
Extra new lines in log output

### ISSUE THREAD

**Issue by ceraolo** [OpenModelica/OpenModelica#10913]
### Description
WHen ouputting log OM adds extra new lines.

### Steps to Reproduce
Run the following code:
```
model Test
  import Modelica.Utilities.Streams.print;
  String sC;
algorithm 
  when initial() then
    for i in 1:3 loop
      sC := "Row" + String(i);
      print(sC);
    end for;
  end when;
end Test;
```
The expected output (and actual output in Dymola) is:
```
Row1
Row2
Row3
```
OM, instead, outputs this:
```
Row1


Row2


Row3
```

This is particularly annoying when we must output text un tabular form (e.g. a matrix)

### Expected Behavior
No extra new lines are added

### Version and OS

 - OpenModelica Version: [e.g. `omc --version` or `Help->About OMEdit` from OMEdit]
 - OS: [e.g. Windows 10, 64 bit]
 - Versions of used Modelica libraries if applicable

**Comment by casella** [comment]
@mahge can you take care of this?

**Comment by adeas31** [comment]
I think OMEdit adds an extra new line.

**Comment by adeas31** [comment]
This works fine for me. Here is the output with `Tools->Options->Simulation->Output->Formatted Text`,

![image](https://github.com/OpenModelica/OpenModelica/assets/4007231/c0fa3f9d-fa6f-40ff-8bec-bd0d47fcc66e)

And here is the output with `Tools->Options->Simulation->Output->Structured`,

![image](https://github.com/OpenModelica/OpenModelica/assets/4007231/0af18ce5-45cd-4b1b-bac1-0e06acf8a3ae)

**Comment by ceraolo** [comment]
I didn't know that there was the option structured/formatted.
In this ticket I used structured, and I confirm that this is the best option when one wants to see the text directly outputted by modelica code.
I saw the issue under OM 1.20.0.

Now I checked with 1.22.0-dev-248 (linux) with structured output and I got the correct result:
![image](https://github.com/OpenModelica/OpenModelica/assets/7373030/df79f7d1-07c6-4813-a1b8-9cf34c8ddda9)

So, I conclude that this issue has been solved between 1.20.0 and 1.22.0-dev-248.
